sdk/communication/AzureCommunicationCalling/Swift_Generated_Interfaces/AzureCommunicationCalling.h

```diff
@@ -0,0 +1,1124 @@
+
+    
+    case unknown = 0
+
+    case groupCallLocator = 1
+
+    case teamsMeetingCoordinatesLocator = 2
+
+    case teamsMeetingLinkLocator = 3
+}
+
+// MARK: Forward declarations.
+
+
+/**
+ * A set of methods that are called by ACSInternalTokenProvider in response to important events.
+ */
+public protocol InternalTokenProviderDelegate : NSObjectProtocol {
+
+    
+    optional func onTokenRequested(_ internalTokenProvider: InternalTokenProvider!, sender: InternalTokenProvider!)
+}
+
+
+/**
+ * A set of methods that are called by ACSCallAgent in response to important events.
+ */
+public protocol CallAgentDelegate : NSObjectProtocol {
+
+    
+    optional func onCallsUpdated(_ callAgent: CallAgent!, args: CallsUpdatedEventArgs!)
+
+    optional func onIncomingCall(_ callAgent: CallAgent!, incomingcall: IncomingCall!)
+}
+
+
+/**
+ * A set of methods that are called by ACSCall in response to important events.
+ */
+public protocol CallDelegate : NSObjectProtocol {
+
+    
+    optional func onIdChanged(_ call: Call!, args: PropertyChangedEventArgs!)
+
+    optional func onStateChanged(_ call: Call!, args: PropertyChangedEventArgs!)
+
+    optional func onRemoteParticipantsUpdated(_ call: Call!, args: ParticipantsUpdatedEventArgs!)
+
+    optional func onLocalVideoStreamsChanged(_ call: Call!, args: LocalVideoStreamsUpdatedEventArgs!)
+
+    optional func onIsRecordingActiveChanged(_ call: Call!, args: PropertyChangedEventArgs!)
+}
+
+
+/**
+ * A set of methods that are called by ACSRemoteParticipant in response to important events.
+ */
+public protocol RemoteParticipantDelegate : NSObjectProtocol {
+
+    
+    optional func onStateChanged(_ remoteParticipant: RemoteParticipant!, args: PropertyChangedEventArgs!)
+
+    optional func onIsMutedChanged(_ remoteParticipant: RemoteParticipant!, args: PropertyChangedEventArgs!)
+
+    optional func onIsSpeakingChanged(_ remoteParticipant: RemoteParticipant!, args: PropertyChangedEventArgs!)
+
+    optional func onDisplayNameChanged(_ remoteParticipant: RemoteParticipant!, args: PropertyChangedEventArgs!)
+
+    optional func onVideoStreamsUpdated(_ remoteParticipant: RemoteParticipant!, args: RemoteVideoStreamsEventArgs!)
+}
+
+
+/**
+ * A set of methods that are called by ACSIncomingCall in response to important events.
+ */
+public protocol IncomingCallDelegate : NSObjectProtocol {
+
+    
+    optional func onCallEnded(_ incomingCall: IncomingCall!, args: PropertyChangedEventArgs!)
+}
+
+
+/**
+ * A set of methods that are called by ACSDeviceManager in response to important events.
+ */
+public protocol DeviceManagerDelegate : NSObjectProtocol {
+
+    
+    optional func onMicrophonesUpdated(_ deviceManager: DeviceManager!, args: AudioDevicesUpdatedEventArgs!)
+
+    optional func onSpeakersUpdated(_ deviceManager: DeviceManager!, args: AudioDevicesUpdatedEventArgs!)
+
+    optional func onCamerasUpdated(_ deviceManager: DeviceManager!, args: VideoDevicesUpdatedEventArgs!)
+}
+
+
```

Comment:
None of these delegate method names align with the guidelines or Apples own delegate methods. To the max extent possible, these should be renamed through annotations.

Reply:
We are trying to be consistent across platforms. Is this considered as a blocker ? I will discuss with the team if we are ok having different names for different platforms.

Reply:
Here's what the SDK Guidelines (general, not just iOS) say regarding consistency:
https://azure.github.io/azure-sdk/general_introduction.html#consistent
```
Client libraries should be consistent within the language, consistent with the service and consistent between all target languages. In cases of conflict, consistency within the language is the highest priority and consistency between all target languages is the lowest priority.
```

Reply:
already marked as GA Blocker, closing this.